app.py

```
@@ -1,5 +1,7 @@
-from flask import Flask, render_template, request, jsonify, session, g
+from flask import Flask, render_template, request, jsonify, session, g, redirect, url_for
```

Ruff (0.12.2)

`flask.redirect` imported but unused

Remove unused import

(F401)